### ISSUE TITLE
[weighting] 11499 has upper cutoffs proportional to energy per nucleon (Trac #1386)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1378]
Set 11499 had CutoffType set to EnergyPerNucleon. If UpperCutoffType is not set, then it defaults to the value of CutoffType, to the primary spectra end at A*1e5 instead of 1e5 as probably intended. Weighting needs to be able to infer that this has happened without breaking properly configured sets.

Migrated from https://code.icecube.wisc.edu/ticket/1386
```json
{
    "status": "closed", 
    "changetime": "2016-04-13T09:10:48", 
    "description": "Set 11499 had CutoffType set to EnergyPerNucleon. If UpperCutoffType is not set, then it defaults to the value of CutoffType, to the primary spectra end at A*1e5 instead of 1e5 as probably intended. Weighting needs to be able to infer that this has happened without breaking properly configured sets.", 
    "reporter": "jvansanten", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1460538648031763", 
    "component": "combo simulation", 
    "summary": "[weighting] 11499 has upper cutoffs proportional to energy per nucleon", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2015-10-06T19:35:38", 
    "milestone": "", 
    "owner": "jvansanten", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20151009T13:21:38: 
* icecube commented:

> This also shows up in the datasets 11058, 11808, 11905, 11926. Dataset 9622 (IC86-I) has no primaries above 1e5GeV in it.


* icecube changed _comment0 from "This also shows up in the datasets 11058, 11808, 11905, 11926." to "1444403341075850"

**Comment by icecube-trac** [comment]
Trac update at 20151009T21:21:55: 
* jvansanten commented:

> In changeset 138425:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="138425"
> Allow CORSIKA sets to have an upper cutoff type, too. This should fix #1378.
> ```


* jvansanten changed status from "new" to "closed"
* jvansanten changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20160413T09:10:48: jvansanten commented:

> In changeset 144662:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="144662"
> Ignore upper cutoff for dCORSIKA. See #1378.
> ```
